src/py/flwr/server/superlink/fleet/vce/backend/raybackend.py

```diff
@@ -160,13 +159,15 @@ async def process_message(
 
             return out_mssg, updated_context
 
-        except LoadClientAppError as load_ex:
+        except Exception as ex:
```

Comment:
I guess we don't care about reporting `LoadClientAppError` back to the SuperLink in the Simulation Engine, right? In that case, the current approach is ok. I just want us to be aware of the semantic difference between Simulation Engine and Deployment Engine.

Reply:
Any potential `LoadClientAppError` will be raised earlier (before starting the actual simulation), we won't be reporting it to the superlink, just trigger a termination of all components involved during a `flower-simulation`.